sources/packages/forms/src/form-definitions/noticeofassessment.json

```diff
@@ -424,7 +424,7 @@
               "value": ""
             }
           ],
-          "content": "Total income used for assessment purposes: <span class=\"label-bold\">{{ utils.custom.currencyFormatter(data.assessment?.totalFamilyIncome) }}</span>",
+          "content": "Total income used for assessment purposes: <span class=\"label-bold\">{{ utils.custom.currencyFormatter(data.assessment?.totalFamilyIncome, \"XXXXX\") }}</span>",
```

Comment:
If `data.assessment?.totalFamilyIncome` is null or undefined, it will display the masked "XXXXX" value for Ministry and even Students, which is not expected.
Since these "masked values" are common, we can incorporate them in the `currencyFormatter` where, if the value `isNaN` but it is "fully masked" with "X", we just return the "X".